config/envs/development.py

```diff
@@ -3,7 +3,6 @@
 from os import getenv
 
 from config.envs.default import DefaultConfig
-from fsd_utils import CommonConfig
```

Comment:
This needs re-adding

Reply:
we're not using this anywhere in the file?

Reply:
lines 23-25 as part of the USE_LOCAL_DATA block i think

Reply:
you're right, have added now 👍🏾